app/views/onboarding/index.html.erb

```diff
@@ -1,36 +1,18 @@
 <%= c 'wrapper', style: :narrow do %>
   <%= c 'box' do %>
-    <%= c 'onboarding_header', logo: '/onboarding/logo.png' %>
+    <%= c 'onboarding_header', logo: Setting.onboarding_logo %>
   <% end %>
 <% end %>
 
 <main>
-  <%= c 'onboarding_hero',
-    photo: '/onboarding/hero.jpg',
-    heading: I18n.t('onboarding.heading')
-  %>
```

Comment:
I don’t really understand why you removed the page title from this component, thus relying on negative margins?

I guess that’s because you wanted to store all the page content’s in a single setting? I’d make an exception for the page title here for multiple reasons:

* I don’t really like applying negative, random margins to elements. In general, I try not to apply margins to components directly, as margins are external styles which in many cases can have unwanted side effects which makes the component context-dependant. Although I think, in this specific case, this might be dismissible, as it’s unlikely that we’re going to reuse the onboarding in other contexts.

* More importantly, I think it makes sense to store the page title separately, as we probably should use it as the HTML document title as well. (This isn’t something we’re doing at the moment, but we should do in the future. See #443)